source/documentation/deploying_apps/deploying_rails.md

```diff
@@ -135,6 +135,26 @@ You can add another instance of your app by running:
 
 ``cf scale APPNAME -i 2``
 
+### Deploying a Rails 5.1+ app with Webpack
+
+Rails 5.1+ allows you to use [webpacker](https://github.com/rails/webpacker) to install and use node.js modules.
+
+You will need to make the following changes to successfully deploy:
+
+1. Specify multiple `buildpacks` in manifest.yml instead of just one, including both the Ruby and node.js buildpacks from GitHub:
+
+    ```yml
+    buildpacks:
+      - https://github.com/cloudfoundry/ruby-buildpack/
+      - https://github.com/cloudfoundry/nodejs-buildpack/
+    ```
+
+1. Exclude `node_modules` during cloudfoundry deployment, which you can do by following the [#excluding-files](https://docs.cloud.service.gov.uk/#excluding-files) instructions and running:
+
+    ```bash
+    ln -s .gitignore .cfignore
+    ```
+
```

Comment:
Is it worth putting in some content about how the user will know when they have successfully deployed a rails 5.1+ app with webpack? Or is it unnecessary because it's the same as deploying a rails app, and users would know what success looks like?